server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -340,7 +324,7 @@ private static boolean assertConsistentEntries(Map<String, List<Entry>> entries)
             final Set<Tuple<String, Integer>> queuedShards = new HashSet<>();
             final List<Entry> entriesForRepository = repoEntries.getValue();
             final String repository = repoEntries.getKey();
-            assert entriesForRepository.isEmpty() == false : "found empty list of snapshots for " + repository + " in " + entries;
+            assert entriesForRepository.isEmpty() == false : "found empty list snapshot snapshots for " + repository + " in " + entries;
```

Comment:
I am not sure this improves the error message. Was this changed intentionally?

Reply:
nah sorry about that, stupid idea renaming bug :)